### PR TITLE
Update otel to 1.0 GA

### PR DESCRIPTION
**Pull request by maciej-szlosarczyk**
Also:
* Removes the need for tracer_id when setting up ecto events
* Always sets library.name parameter to mv_opentelemetry.

### CODE REVIEW

**Review by feliperenan — APPROVED** [review]
LGTM 👍🏻

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: ad5083fc61464f6eab4cf9c237f286b79902dc09</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/17#pullrequestreview-847794775" title="Approved">feliperenan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [feliperenan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
